moveit_setup_assistant/templates/moveit_config_pkg_template/launch/demo_gazebo.launch

```diff
@@ -58,7 +58,7 @@
 
   <!-- Run Rviz and load the default config to see the state of the move_group node -->
   <include file="$(find [GENERATED_PACKAGE_NAME])/launch/moveit_rviz.launch">
-    <arg name="config" value="true"/>
+    <arg name="rviz_config" value="true"/>
```

Comment:
The new argument requires a config file instead of boolean value. See #1397.
```suggestion
    <arg name="rviz_config" value="$(find [GENERATED_PACKAGE_NAME])/launch/moveit.rviz"/>
```